src/entity-manager/EntityManager.ts

```diff
@@ -178,7 +180,9 @@ export class EntityManager extends BaseEntityManager {
         } else {
             // todo: throw exception if constructor in target is not set
             if (target instanceof Array) {
-                return this.getRepository<Entity[]>(target[0].constructor).remove(entity as Entity[], options);
+                return Promise.all(target.map((t, i) => {
```

Comment:
i know its a short peace of code, but please prefer longer and more clear code in such places as well